### PR TITLE
Small clean up for packaging

### PR DESCRIPTION
**Pull request by ericdill**
attn @arkilic

### CODE REVIEW

**Comment by ericdill** [comment]
@tacaswell can we drop the from __future__ imports if this is py3.4+ only library?

File: amostra/client/amutils.py [hunk 1]
```diff
@@ -17,9 +16,10 @@ def doc_or_uid_to_uid(doc_or_uid):
     uid : str
         A string version of the uid of the given document
     """
-    if not isinstance(doc_or_uid, six.string_types):
-        doc_or_uid = doc_or_uid['uid']
-    return str(doc_or_uid)
+    try:
+        return str(doc_or_uid['uid'])
+    except KeyError:
```
Comment (arkilic) [inline amostra/client/amutils.py]: This should be a TypeError in addition to KeyError bc if uid is a string, it complains string indices must be ints.

Comment (ericdill) [inline amostra/client/amutils.py]: Yeah I just saw that. Good catch

**Comment by ericdill** [comment]
Ok this should be good to go, unless you think we should keep six and `from __future__`

**Comment by arkilic** [comment]
Thanks @ericdill

**Pull request merged**